clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_pipeline.py

```diff
@@ -21,54 +21,25 @@ class DwiPreprocessingUsingT1(cpe.Pipeline):
     """DWI Preprocessing using T1 image for susceptibility distortion step.
 
     Warnings:
-        - Do not use this pipelines if you have fieldmap data in your dataset.
-
-    Args:
-        input_dir(str): Input directory in a BIDS hierarchy.
-        output_dir(str): Output directory in a CAPS hierarchy.
-        subjects_sessions_list(str): The Subjects-Sessions list file (in .tsv
-            format).
+        - Do not use this pipeline if you have fieldmap data in your dataset.
 
     Returns:
         A clinica pipeline object containing the DWIPreprocessingUsingT1 pipeline.
 
     Raises:
 
-
     """
-    def __init__(self,
-                 bids_directory=None,
-                 caps_directory=None,
-                 tsv_file=None,
-                 base_dir=None,
-                 name=None,
-                 low_bval=5):
-        """
-
-        Args:
-            low_bval (int): Define the b0 volumes as all volume
-                bval <= lowbval. (Default = 5)
-        """
+    def check_pipeline_parameters(self):
+        """Check pipeline parameters."""
         import warnings
 
-        super(DwiPreprocessingUsingT1, self).__init__(
-            bids_directory=bids_directory,
-            caps_directory=caps_directory,
-            tsv_file=tsv_file,
-            base_dir=base_dir,
-            name=name)
-
-        self._low_bval = low_bval
-
-        if self._low_bval < 0:
-            raise ValueError('The low_bval is equals to '
-                             + str(self._low_bval)
-                             + ': it should be zero or close to zero.')
+        if self.parameters['low_bval'] < 0:
+            raise ValueError(
+                'The low_bval is equals to %s: it should be zero or close to zero.' % self.parameters['low_bval'])
 
-        if self._low_bval > 100:
-            warnings.warn('Warning: The low_bval parameter is ('
-                          + str(self._low_bval)
-                          + '), it should be close to zero', UserWarning)
+        if self.parameters['low_bval'] > 100:
+            warnings.warn('Warning: The low_bval parameter is %s: it should be close to zero' % self.parameters['low_bval'],
```

Comment:
> Good work, i think it's almost ready, only had a few comments. But for the future, this is not good to have PR that are that long, there is way too much code to handle in a single review of code. 66 modified files is way too much, and tons of things happen at the same time when reviewing, making it difficult to grasp the big picture.

Sorry about that. When I finished pipeline harmonization, I should have start a new PR regarding `{t1|pet}-volume` pipelines. It would have lessen to ~40 files + ~30 files.

Reply:
> Why use warning methods instead of print/cprint ? This is not done anywhere else in Clinica (at least not until a logger is integrated)

I forgot to adress this comment. This code was present "long" time before `cprint`. This is now replaced in 3924b6e.